### PR TITLE
Improve application confirm

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: src/application-confirm/index.jsx [hunk 1]
```diff
@@ -1,28 +1,56 @@
...
+      <div className="application-confirm">
+        <CheckboxGroup
+          name="declarations"
+          label={<Snippet>declarations.title</Snippet>}
+          options={this.props.schema.declarations.options}
+          error={this.props.error}
+          onChange={this.onChange.bind(this)}
+          value={this.state ? this.state.values : []}
+        />
+        <button type="submit" className="govuk-button" disabled={!this.state || this.state.submitDisabled}>
```
Comment (lennym) [inline src/application-confirm/index.jsx]: This looks like the default state will be with the button disabled - ideally the default with no state should be enabled  and only disabled on the client when the state is created.

**Comment by lennym** [comment]
Will this still do validation of the checkboxes on the server?

File: src/application-confirm/index.jsx (hunk 1 above)
Comment (wheelsandcogs) [reply src/application-confirm/index.jsx]: Done.

**Comment by wheelsandcogs** [comment]
Server-side validation is in place if you use the form router and provide the schema correctly when using this component.

**Pull request merged**